lib/contentful-util.js

```diff
@@ -62,22 +62,27 @@ function normalizeEntry({ contentTypes, entry, options }) {
     return Object.assign({}, fields, { __metadata: metadata });
 }
 
-function resolveLinks(entry, assets, entries) {
-    if (!entry.fields) return entry;
+function resolveLink(value, assets, entries) {
+    if (Array.isArray(value)) {
+        return value.map(childValue => resolveLink(childValue, assets, entries));
+    }
 
-    const resolvedFields = Object.keys(entry.fields).reduce((result, fieldName) => {
-        const localeKey = Object.keys(entry.fields[fieldName])[0];
+    if (value.sys && value.sys.type === 'Link') {
```

Comment:
```suggestion
    if (value && value.sys && value.sys.type === 'Link') {
```

Reply:
@eduardoboucas sorry I missed that,
There should be check for `value`.

Reply:
Already taken care of and released! 🚀